3-semantic-html/index.html

```diff
@@ -0,0 +1,30 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Html Semantico</title>
+</head>
+<body>
+    <header>
+        <div>h1</div>
+          <nav>Navegación</nav>
+        </header>
+        
+        <main>
+        <div>article</div>
+        <div>section</div>
+        </main>
```

Comment:
```suggestion
        <main>
        <article>
        <header>
        <h1>Content</h1>
        </header>
        <section>
        <p>content</p>
        </section>
        <footer>content</footer>
        </article>
        </main>
```
De acuerdo a la imagen indicada en el ejercicio, los contenidos dentro de la etiqueta `<main>`, son definidos por ésta imagen: https://user-images.githubusercontent.com/36536646/79254810-e69dc380-7e4a-11ea-9f5e-75c6b8efec0b.png